lib/plugins/newrelicTransactionManagerPlugin.ts

```diff
@@ -50,6 +51,17 @@ export class NewRelicTransactionManager implements TransactionObservabilityManag
     newrelic.addCustomAttribute(attrName, attrValue)
   }
 
+  public addCustomAttributes(
+    _uniqueTransactionKey: string,
```

Comment:
signature of the method still includes this parameter because other implementations (Prometheus, OpenTelemetry etc) will need it